config/dev.exs

```diff
@@ -78,7 +78,8 @@ config :screenplay,
   sftp_client_module: Screenplay.Outfront.FakeSFTPClient,
   config_fetcher: Screenplay.Config.LocalFetch,
   local_config_file_spec: {:priv, "places_and_screens.json"},
-  local_locations_file_spec: {:priv, "screen_locations.json"}
+  local_locations_file_spec: {:priv, "screen_locations.json"},
+  local_descriptions_file_spec: {:priv, "place_descriptions.json"}
```

Comment:
can this be renamed to something that indicates "place": maybe `local_place_descrip_file`?